templates/core/about_us.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
please take a look at how the new pages are done, we have a layout template. These pages will now need to utilise them like so:  https://github.com/PolyglotDevsLondon/sharespots/blob/develop/templates/core/index.html

Reply:
Ok sure thing..